core/actions/assertion.ts

```diff
@@ -274,6 +272,21 @@ export class Assertion extends ActionBuilder<dataform.Assertion> {
     return dataform.Target.create(this.proto.target);
   }
 
+  /** @hidden */
+  public getParentAction() {
+    return dataform.Target.create(this.proto.parentAction);
+  }
+
+  /** @hidden */
+  public setParentAction(target: dataform.Target) {
+    return (this.proto.parentAction = target);
```

Comment:
Why do we need returns on setters? My expectations that it just sets the value.

The same question about other setters